travis-ci-prod-services-1/firewall.tf

```diff
@@ -0,0 +1,14 @@
+resource "google_compute_firewall" "cert-manager-webhook-allow" {
+  name    = "cert-manager-webhook-allow"
+  network = "${module.networking.main_network_name}"
+  project = "${module.project.project_id}"
+
+  source_ranges = ["172.16.0.0/28"]
+
+  allow {
+    protocol = "tcp"
+    ports    = ["6443"]
+  }
+
+  source_tags = ["services"]
```

Comment:
Does it need both `source_ranges` and `source_tags`? We could simply use `source_tags` and fill it in with a variable that is also used on the cluster's node pool. In case we then decide to change the tag, it changes on both locations at once.

Reply:
source_ranges is needed